artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java

```diff
@@ -344,4 +344,7 @@ public interface ActiveMQClientLogger {
 
    @LogMessage(id = 214033, value = "Cannot resolve host ", level = LogMessage.Level.ERROR)
    void unableToResolveHost(UnknownHostException e);
+
+   @LogMessage(id = 214034, value = "{} is being garbage collected without removing its resources.:\nlocations:\n{}", level = LogMessage.Level.INFO)
+   void debugMessageNotReleased(String clazz, String debugString);
```

Comment:
RefCountMessage is at Client.

Reply:
I tried moving CoreMessage to the server, but that would mean having to send the Server jar to the clients...


I have been there before. this was the best compromise some time ago.

Reply:
I hadnt actually noticed that those methods are defined in the Message interface, since RefCountMessage doesnt actually indicate it implements anything itself...but things subclass it to effectively implement the Message methods, which makes it rather hard to see whats using it and what the original impl is when the Message method is being overridden in various places.